src/FixedDataTable.js

```diff
@@ -184,6 +184,24 @@ var FixedDataTable = createReactClass({
     rowHeightGetter: PropTypes.func,
 
     /**
+     * Pixel height of cells unless `cellHeightGetter` is specified and returns
+     * different value.
+     */
+    cellHeight: PropTypes.number,
```

Comment:
I'm not sure on the API here.  It's more intuitive to me to pass a rowHeight + an expandedContentHeight.  That would require more logic in the scroll helper / system to combine them there though.  Any thoughts on making that change?